### PR TITLE
improve accessibility in LOBSTER html report

### PR DESCRIPTION
**Pull request by ankushsonare1**
improve accessibility in LOBSTER report: HTML report #26

As a maintainer of the LOBSTER tool

I change prominent colours.

So that visually impaired people (color blindness) can distinguish easily between error and success

### CODE REVIEW

**Comment by mugdhadhole1** [comment]
Colors seems ok
Ok to merge

**Comment by florianschanda** [comment]
I suggest to add a command-line flag to do this, or even a javascript enabled toggle in the document itself.

Changing the default colours for everyone is not intuitive.

**Comment by florianschanda** [comment]
I suggest to add a command-line flag to do this, or even a javascript enabled toggle in the document itself.

Changing the default colours for everyone is not intuitive.

**Review by christophkloeffel — CHANGES_REQUESTED** [review]
please remove report.lobster and lobster_report.html

File: report.lobster [hunk 1]
```diff
@@ -0,0 +1,13353 @@
+{
```
Comment (phiwuu) [inline report.lobster:1]: Please don't add this file. I guess it is a mistake?

File: lobster_report.html [hunk 2]
```diff
@@ -0,0 +1,11218 @@
+<!DOCTYPE html>
```
Comment (phiwuu) [inline lobster_report.html:1]: Please don't add this file. I guess it is a mistake?

**Review by phiwuu — CHANGES_REQUESTED** [review]

**Comment by phiwuu** [comment]
The new images look very nice.

**Comment by phiwuu** [comment]
Please clarify the copyright of the new icons. If we are OK to use them, then please update the colours as follows:

1. use a darker green instead of light green, e.g. `b2e1b2`
2. change red to purple, e.g. `ffb2ff`
3. keep the gray as is

**Review by florianschanda — DISMISSED** [review]
I suggest to add a command-line flag to do this, or even a javascript enabled toggle in the document itself.

Changing the default colours for everyone is not intuitive.

**Comment by phiwuu** [comment]
I saw your comment, but I don't get the point. What do you mean with "not intuitive"?
The color changes are minimal, but solve the issue for visually impaired people. Adding a command-line flag or javascript is much more effort compared to changing the colors. We want to close this issue and move on.

**Comment by ankushsonare1** [comment]
I have check new svg icons its not copyright.

**Comment by florianschanda** [comment]
Traffic light colours are universally clear.
The proposed colours are not - you need to add a legend at the very least.

**Comment by florianschanda** [comment]
I would still strongly argue for a command-line option if a live, in document option using JS is too hard to do. It won't be too hard to do given how we bake in the style sheet anyway.

File: lobster/tools/core/html_report.py [hunk 3]
```diff
@@ -286,13 +286,13 @@ def write_html(fd, report, dot):
         "padding-left"  : "0.2em",
     }
     doc.style[".item-ok"] = {
-        "background-color" : "#efe",
+        "background-color" : "#b2e1b2",
```
Comment (phiwuu) [inline lobster/tools/core/html_report.py:289]: please add a switch case here, as suggested by @florianschanda

File: lobster/tools/core/html_report.py [hunk 4]
```diff
@@ -286,13 +286,13 @@ def write_html(fd, report, dot):
...
     }
     doc.style[".item-partial"] = {
         "background-color" : "#ffe",
     }
     doc.style[".item-missing"] = {
-        "background-color" : "#fee",
+        "background-color" : "#ffb2ff",
```
Comment (phiwuu) [inline lobster/tools/core/html_report.py:295]: and here as well

**Review by phiwuu — CHANGES_REQUESTED** [review]

File: lobster/tools/core/html_report.py [hunk 5]
```diff
@@ -465,6 +465,9 @@ def main():
                     help="path to dot utility (https://graphviz.org), \
                     by default expected in PATH",
                     default=None)
+    ap.add_argument("--color_blind",
```
Comment (christophkloeffel) [inline lobster/tools/core/html_report.py:468]: please use flag `--color-blind`

File: lobster/tools/core/html_report.py [hunk 6]
```diff
@@ -465,6 +465,9 @@ def main():
...
+                        action="store_true",
+                        help="Enable color blind mode")
```
Comment (christophkloeffel) [inline lobster/tools/core/html_report.py:470]: be more specific in the help message: 
something like "uses a color palette with a higher contrast"

**Review by christophkloeffel — CHANGES_REQUESTED** [review]

**Comment by christophkloeffel** [comment]
please also create an entry in the changelog

**Review by christophkloeffel — APPROVED** [review]

**Review by phiwuu — APPROVED** [review]

**Pull request merged**